plugin.go

```diff
@@ -19,10 +19,8 @@ var (
 
 // DeviceIdentifier defines the AMT-specific way of uniquely identifying a device
 // through its device configuration.
-//
-// FIXME - this is just a stub for framing up the plugin
 func DeviceIdentifier(data map[string]string) string {
-	return ""
+	return data["ip"]
```

Comment:
This is okay for now, but there is a bit of an issue to watch out for in that the IP is not actually unique. All of the devices will have the same IP. 

This ends up being okay here because we should only ever have one power and one boot target device. If, however, we were to have >1 of either device, they would end up having the same ID. I've been thinking through this issue for the next rev of the SDK, but haven't settled on what the best approach for cases like this are.